test_coordinates

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Ok sweet no big deal lol
On May 2, 2016 12:42 PM, "Jake Guymon" <notifications@github.com> wrote:

> In test_coordinates
> <https://github.com/kaorus-army/chess_with_friends/pull/5#discussion_r61764523>
> :
>
> > @@ -0,0 +1,26 @@
> > +
>
> Oh this is a mistake. I was trying to check what was going in to
> test_coordinates using pry but it wasn't working correctly for whatever
> reason and i guess I made a log of it. This is just garbage, ill delete it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaorus-army/chess_with_friends/pull/5/files/09d78761baed34dd6817e2a38db66aefb5dd5bd0#r61764523>
>